backend/src/apiserver/main.go

```diff
@@ -72,12 +80,47 @@ func main() {
 		template.Launcher = common.GetStringConfig(launcherEnv)
 	}
 
-	clientManager := cm.NewClientManager()
+	logLevel := *logLevelFlag
+	if logLevel == "" {
+		logLevel = "info"
+	}
+
+	var zapLevel zapcore.Level
+	switch logLevel {
+	case "debug":
+		zapLevel = zapcore.DebugLevel
+	case "info":
+		zapLevel = zapcore.InfoLevel
+	case "warn":
+		zapLevel = zapcore.WarnLevel
+	case "error":
+		zapLevel = zapcore.ErrorLevel
+	case "dpanic":
+		zapLevel = zapcore.DPanicLevel
+	case "panic":
+		zapLevel = zapcore.PanicLevel
+	case "fatal":
+		zapLevel = zapcore.FatalLevel
+	default:
+		glog.Fatalf("Invalid log level: %s", logLevel)
```

Comment:
Perhaps default to info here would be safer in case there is a log level missed in the swtich.